config/initializers/slim.rb

```diff
@@ -1,5 +1,5 @@
 # Slim
 Slim::Engine.set_options format: ENV['html_format'].to_sym,
-                         pretty: ENV['html_pretty'] == 'pretty',
+                         pretty: (ENV['html_pretty'] == 'true'),
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.